storagenode/storagenodedb/reputation.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+import (
+	"context"
+
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode/reputation"
+)
+
+// Reputation returns reputation.DB
+func (db *InfoDB) Reputation() reputation.DB { return &reputationDB{db} }
+
+// Reputation returns reputation.DB
+func (db *DB) Reputation() reputation.DB { return db.info.Reputation() }
+
+// reputation works with node reputation DB
+type reputationDB struct {
+	*InfoDB
+}
+
+// Store inserts or updates reputation stats into the db
+func (db *reputationDB) Store(ctx context.Context, stats reputation.Stats) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	stmt := `INSERT OR REPLACE INTO reputation (
```

Comment:
How can this every be an Update/Replace?

Reply:
Same timestamp/satellite

Reply:
That should never happen. Else we have a bug in our software!

Reply:
Oh, sorry, that's reputation, so I am just replacing satellite old value